src/_locales/ru/messages.json

```diff
@@ -17,10 +17,10 @@
     "js_context_toggle_pause_suspension": { "message": "Включить/выключить приостановку этой вкладки" },
     "js_context_never_suspend_domain": { "message": "Никогда не приостанавливать этот домен" },
     "js_context_never_suspend_page": { "message": "Никогда не приостанавливать этот URL" },
-    "js_context_suspend_selected_tabs": { "message": "Приостановать выбранные вкладки" },
+    "js_context_suspend_selected_tabs": { "message": "Приостановить выбранные вкладки" },
```

Comment:
How does this context menu item work?

Reply:
now I understand, and in this case it would be necessary to remove these items from the context menu rmb

Reply:
Well u may create an issue, sounds reasonable, it would be interesting to know dev's opinion on this topic and why these items there.

Reply:
Suspending selected tabs via the rightclick context menu works fine for me?
If I select multiple tabs (by holding down cmd and clicking the tab favicons) and then right click somewhere on the page of the currently focused tab, I get the TGS context menu. This does NOT unselect my tabs. And when I choose 'Suspend selected tabs' this suspends all the tabs I had highlighted.

Reply:
@deanoemcke I assume it maybe coz u r on macOS and we r on Win 10? Maybe Chrome behaves differently for us here? 🤔 And even if it does I would say there's no need to remove these options then if they work with other OSes, even checking OS to enable may be too much of a hassle then.

Reply:
thanks for the answers, at least the translation question is closed